app/controllers/leave_applications_controller.rb

```diff
@@ -1,4 +1,24 @@
 class LeaveApplicationsController < BaseController
   def index
   end
+
+  def show
```

Comment:
你不是還在玩貓嗎？
怎麼一轉眼就在看扣了 XDD

Tsehau Chao <notifications@github.com> 於 2016年7月27日 週三 下午6:09寫道：

> In app/controllers/leave_applications_controller.rb
> <https://github.com/jodeci/daikichi/pull/5#discussion_r72413779>:
>
> > @@ -1,4 +1,24 @@
> >  class LeaveApplicationsController < BaseController
> >    def index
> >    end
> > +
> > +  def show
>
> show 不用另外定義喔
> collection_scope 已指定當前物件
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jodeci/daikichi/pull/5/files/c6cc7d687376764397a93c8556b37fcab4f581bc#r72413779>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIm9T4sYVM5nGdJBhAUIm0WoLeiLVjE5ks5qZy5fgaJpZM4JV-zL>
> .
>